winevdm.txt

```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = otvdm
+Description = Enables support for wow16 applications
```

Comment:
```suggestion
Description = Enables support for WOW16 applications. Extract in Program Files directory.
```

Reply:
I didn't tested the installer yet, but isn't it possible to just always extract to Program Files, so user won't be able to do a mistake here?

Reply:
Indeed. You're right. I forgot that it's already a custom installer, so we can change it as we need. Then this addition seems not required. :slightly_smiling_face: